hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/statistics/IOStatisticsLogging.java

```diff
@@ -249,6 +250,26 @@ public static void logIOStatisticsAtDebug(
     logIOStatisticsAtDebug(LOG, message, source);
   }
 
+  /**
+   * A method to log IOStatistics from a source.
+   *
+   * @param log    Logger for logging.
+   * @param level  LOG level.
+   * @param source Source to LOG.
+   */
+  public static void loggingIOStatistics(Logger log, String level,
+      Object source) {
+    if (level.toLowerCase().equals(IOSTATISTICS_LOGGING_LEVEL_INFO)) {
```

Comment:
proposed: use a switch() and support error and warn too; the lower case locale must be in EN_US. (trivia: the code as is won't work in turkey).